### DISCUSSION TITLE
Renaming issue of an old organization in NPM.

### DISCUSSION THREAD

**Discussion by ernstchristian1202** [community/community#143758] [npm]
I created an organization with an incorrect name in NPM account.
I need to change the name but I dont know how.
Would you please tell me how I can do it?

**Comment by radanpro** [comment]
### I think you should contact NPM support and explain the issue to them. They will be able to help you change the organization name.
😊

**Reply by ernstchristian1202** [reply]
Thanks.
I already asked tech team on twitter.
But didn't yet any answer.

**Reply by radanpro** [reply]
I think they will respond to you here. If they don't, try asking on their official site: 
[NPM Support Page](https://www.npmjs.com/support)
 😊
